.github/workflows/build.yml

```diff
@@ -196,6 +196,29 @@ jobs:
         run: |
           .\MNE-Python-%MNE_INSTALLER_VERSION%-Windows.exe /S /InstallationType=JustMe /AddToPath=1
 
+      - name: Export frozen environment definition (macOS)
```

Comment:
This whole block is verbatim copy-paste just to make sure the `.json` is available before the testing blocks